tests/validate-sigma-schema/sigma-schema.json

```diff
@@ -217,11 +221,13 @@
       ]
     },
     "tags": {
+      "description": "Tags to categorize a Sigma rule.",
       "type": "array",
+      "uniqueItems": true,
       "items": {
         "type": "string",
-        "pattern": "^[a-z0-9_.-]+$"
+        "pattern": "^[a-z0-9_-]+\\.[a-z0-9._-]+$"
```

Comment:
Can you double check that this covers the tags in the xxxx.whatever ? @phantinuss

Reply:
I don't think tags need to be of that form.

Reply:
The spec is also very clear on the allowed character set. We should add "numerals" explicitly to that sentence, because it's only allowed by examples right now. Will do a PR: https://github.com/SigmaHQ/sigma-specification/pull/105